conduct.md

```diff
@@ -72,13 +72,15 @@ TODO: Write an introductory paragraph.
 
 #### Acceptable Conduct
 
-- TODO
+- Create decision that mutually benefit each other (Schedules, Coding). This is to help understand each other's responsibilities and respect on what is possible for both individual interest and the team interest.
```

Comment:
You may want to reword the first sentence to improve the clarity of the conduct outlined. Maybe something along the lines of 'communicate with each other to create mutually beneficial decisions in regard to scheduling and coding commitments.'

Reply:
Was written by another team that already merged to main--content was pulled in when branch was created